draft-ietf-rats-eat.md

```diff
@@ -288,54 +288,40 @@ An entity may have strong security defenses against hardware invasive attacks.
 It may also have low security, having no special security defenses.
 There is no minimum security requirement to be an entity.
 
-## CWT, JWT and Detached EAT Bundle
 
-An EAT is primarily a claims set about an entity based on one of the following:
+## EAT as a Framework
 
-* CBOR Web Token (CWT) {{RFC8392}}
-* JSON Web Token (JWT) {{RFC7519}}
+EAT can be viewed as a framework for defining attestation tokens for specific use cases more so than it is one specific token definition.
+While EAT is based on and compatible with CWT and JWT, it can be thought of in these terms:
 
-All definitions, requirements, creation and validation procedures, security considerations, IANA registrations and so on from these carry over to EAT.
+* An identification and type system for claims in claims-sets
+* Definitions of common attestation-oriented claims
+* Claims are defined in CDDL and serialized using CBOR or JSON
+* Security envelopes based on COSE and JOSE
+* Nesting of claims sets and tokens to represent complex and compound devices
+* A profile mechanism for specifying and identifying specific token formats for specific use cases
 
-This specification extends those specifications by defining additional claims for attestation.
-This specification also describes the notion of a "eat_profile" that can narrow the definition of an EAT, ensure interoperability and fill in details for specific usage scenarios.
-This specification also adds some considerations for registration of future EAT-related claims.
+EAT uses the name/value pairs the same as CWT and JWT to identify individual claims.
+{{theclaims}} defines common attestation-oriented claims that are added to the CWT and JWT IANA registries.
+As with CWT and JWT, no claims are mandatory and claims not recognized should be ignored.
 
-The identification of a protocol element as an EAT, whether CBOR or JSON encoded, follows the general conventions used by CWT, JWT.
-Largely this depends on the protocol carrying the EAT.
-In some cases it may be by content type (e.g., MIME type).
-In other cases it may be through use of CBOR tags.
-There is no fixed mechanism across all use cases.
+Unlike, but compatible with CWT and JWT, EAT defines claims using Concise Data Definition Language (CDDL) {{RFC8610}}.
+In most cases the same CDDL definition is used for both the CBOR/CWT serialization and the JSON/JWT serialization.
```

Comment:
Maybe summarize the differences here?

Reply:
Other people think this section is too long.